subprojects/packagefiles/rdkafka/meson.build

```diff
@@ -103,9 +103,8 @@ curl = dependency('libcurl', required: get_option('WITH_CURL'))
 zlib = dependency('zlib', required: get_option('WITH_ZLIB'))
 zstd = dependency('libzstd', required: get_option('WITH_ZSTD'))
 lz4 = dependency('liblz4', required: get_option('WITH_LZ4_EXT'))
-sasl = dependency('libsasl2', required: get_option('WITH_SASL')) # TODO: make SASL usable without libsasl2 on windows 
-ssl = dependency('libssl', required: get_option('WITH_SSL'))
-crypto = dependency('libcrypto', required: get_option('WITH_SSL'))
+sasl = dependency('libsasl2', required: get_option('WITH_SASL')) # TODO: make SASL usable without libsasl2 on windows
+ssl = dependency('openssl', required: get_option('WITH_SSL'), method: 'pkg-config')
```

Comment:
This is wrong...

The actual problem is Strawberry Perl is terrible and should be rm -rf'ed from the runner VM image, and the latest cmake trips over that broken installation.

Reply:
No argument there.